integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestRenameTableWithIndex.scala

```diff
@@ -107,6 +107,17 @@ class TestRenameTableWithIndex extends QueryTest with BeforeAndAfterAll {
       s"""
          | explain select * from carbon_tb where name='eason'
        """.stripMargin).collect()
+
+    sql(
+      s"""
+         | CREATE INDEX dm_carbon_si
+         | ON TABLE carbon_tb (name, city)
+         | AS 'carbondata'
+      """.stripMargin)
+
+    sql(s"""alter TABLE carbon_tb rename to carbon_table""".stripMargin)
```

Comment:
as this operation is already done on line:93, you can move create si index before and validate

Reply:
done